covsirphy/cleaning/pcr_data.py

```diff
@@ -0,0 +1,439 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import numpy as np
+import pandas as pd
+from dask import dataframe as dd
+from covsirphy.util.plotting import line_plot
+from covsirphy.util.error import SubsetNotFoundError
+from covsirphy.cleaning.cbase import CleaningBase
+from covsirphy.cleaning.country_data import CountryData
+
+
+class PCRData(CleaningBase):
+    """
+    Data cleaning of PCR dataset.
+
+    Args:
+        filename (str): CSV filename of the dataset
+        citation (str): citation
+    """
+    # Column names
+    TESTS = "Tests"
+    TESTS_JPN = "Tested"
```

Comment:
Yes, "Tests" is the appropriate term because this is not the number of cases. In another pull request, I will change "Tested" with "Tests" in Japan dataset and update `PCRData" class.

Reply:
Reslove this comment and I will update in my pull request.